src/routes/(app)/docs/[lang]/+server.ts

```diff
@@ -0,0 +1,14 @@
+import fs from 'fs';
+import path from 'path';
+
+export async function GET({ params }) {
+	const { lang } = params;
+	const docsPath = path.resolve('docs', lang);
+
+	try {
+		const files = fs.readdirSync(docsPath).filter((file) => file.endsWith('.md'));
+		return new Response(JSON.stringify(files), { status: 200 });
+	} catch (error) {
+		return new Response(JSON.stringify({ error: 'Folder not found' }), { status: 404 });
```

Comment:
use the `json` sveltekit elper or make sure to add the `Content-Type: application/json` header